packages/desktopjs-openfin/src/openfin.ts

```diff
@@ -573,7 +539,7 @@ export class OpenFinContainer extends WebContainerBase {
     }
 
     public showNotification(title: string, options?: NotificationOptions) {
```

Comment:
same as grouping methods, the base class already throws not implemented so you can just delete the whole member